src/card.scss

```diff
@@ -41,6 +42,13 @@ $fd-card-content-avatar-text-spacing: 0.5rem !default;
     @include fd-reset();
     @include fd-flex();
 
+    padding: 1rem;
+    background: $fd-card-default-body-background-color;
+    border-bottom: $fd-card-header-border;
+    border-radius: $fd-card-header-border-radius $fd-card-header-border-radius 0 0;
+    text-decoration: none;
+    cursor: pointer;
+
```

Comment:
why did you move these rules here?
not that it's a big deal but usually all @include statements are at the beginning (except rtl) and after that come the normal css rules

Reply:
It's done because it is totally `fd-hover/active/selected` are totally different mixins than `flex/reset`. I treat them same as `&:hover/active/selected`. That's why those properties have been moved up

Reply:
of course they are different. But the order of the css should not be important :) it's just a common practice in scss world that the include statements are at the beginning.